package.json

```diff
@@ -60,8 +60,8 @@
     "lodash": "4.17.4",
     "merge": "^1.2.0",
     "ng2-charts": "1.6.0",
-    "ng2-tag-input": "1.2.8",
     "ngx-bootstrap": "1.6.6",
+    "ngx-chips": "^1.4.5",
```

Comment:
Any reason we aren't using patch semver? I know yarn should handle this but I guess I'm traumatized from past experiences..